recipes-core/images/intel-aero-image.bb

```diff
@@ -35,6 +35,7 @@ IMAGE_INSTALL += "rng-tools"
 
 # Build tools
 IMAGE_INSTALL += "packagegroup-core-buildessential"
+IMAGE_INSTALL += "catkin"
```

Comment:
Have anybody been able to follow up on this issue with this pull request (the missing prefix path)?